flake-module.nix

```diff
@@ -228,7 +228,7 @@ ihpFlake:
                     fi
 
                     ${pkgs.nixos-rebuild}/bin/nixos-rebuild switch -j auto --use-substitutes --fast --flake .#$1 --target-host $1 --build-host $1 --option substituters https://digitallyinduced.cachix.org --option trusted-public-keys digitallyinduced.cachix.org:digitallyinduced.cachix.org-1:y+wQvrnxQ+PdEsCt91rmvv39qRCYzEgGQaldK26hCKE=
```

Comment:
How to invoke this from the project root by hand? I'd like to debug the error of `error: invalid character in Base64 string: ''` that likely comes from here.
I tried `nixos-rebuild` directly, and `nix-shell --run nixos-rebuild`, none of these work.